File: java/src/test/java/BasicTest.java
import org.junit.Assert;
import org.junit.Test;

public class BasicTest extends TestCase {

    @Test
    public void visitSwagLabs() {
        driver.get("https://www.saucedemo.com/v1/");

        Assert.assertTrue(driver.getTitle().contains("Swag"));
    }
}

File: kotlin/src/test/kotlin/BasicTest.kt
import org.junit.jupiter.api.*


@TestInstance(TestInstance.Lifecycle.PER_CLASS)
class BasicTest() : TestCase() {

    @Test
    fun visitSwagLabs() {
        driver.get("https://www.saucedemo.com/v1/")

        Assertions.assertTrue(("Swag") in driver.title, "Incorrect page title")
    }
}